lib/typing/columns/diff.go

```diff
@@ -23,12 +26,16 @@ func shouldSkipColumn(colName string, softDelete bool, includeArtieUpdatedAt boo
 		return false
 	}
 
+	if mode == config.History && colName == constants.OperationColumnMarker {
```

Comment:
Nit: The other three checks compare `colName` first, swap comparison for consistency.